apps/frontend/src/libs/components/select/libs/constants/select-class-names.constant.ts

```diff
@@ -0,0 +1,27 @@
+import { combineClassNames } from "~/libs/helpers/helpers.js";
+
+import styles from "../../styles.module.css";
+
+const selectClassNames = {
```

Comment:
By conventions in style guide we need to name objects in the singular form, don't forget to update file name
```suggestion
const selectStylesConfig = {
```